.vale/styles/RedHat/Spelling.yml

```diff
@@ -485,6 +485,7 @@ filters:
   - Webpack
   - WebSocket
   - WebView
+  - wifi
```

Comment:
Should we add this as a case-insensitive entry on line 221 instead?
```suggestion
  - "[wW]ifi"
```

Reply:
Actually Wifi isn't allowed per IBM style guide. Basically, it's Wi-Fi or wifi. However, I guess Wifi could appear at the start of a sentence.... 

![image](https://github.com/redhat-documentation/vale-at-red-hat/assets/104497497/aa409e23-c287-43ab-a2f9-fe78bc27c3b8)

Reply:
I agree, @rohennes: `  - "[wW]ifi"` is going to mislead users more often and more significantly that `  - wifi`. So let's stick with the latter.